### ISSUE TITLE
Question about validating collections

### ISSUE THREAD

**Issue by mt0erfztxt** [friemen/examine#2]
Hi, I have a question about validating collections.
Let's take some data and create validators for it
```clojure
(def person {:contacts [{:name "Donald", :age "40"} {:name "Mickey", :age false}]})
(def contact-rules (rule-set :age required is-integer))
(def person-rules (rule-set :contacts (for-each (partial validate contact-rules))))
```
Now we can validate our `person`
```clojure
(messages (validate person-rules person))
;; {[0 :age] ("Must be an integer number"), [1 :age] ("Must be an integer number")}
```
as a result we have validation errors as expected, but path in each message isn't complete as its first element, which is a `:contacts`, is missing and my question is "Why it's missing?".

**Comment by friemen** [comment]
I reproduced your use-case as unit-test and it seems like a bug which occurs when collections are combined with nested validation.
I'll try to fix it ASAP.

**Comment by friemen** [comment]
Fixed.
Released version 1.3.1 to https://clojars.org/examine

**Comment by mt0erfztxt** [comment]
@friemen Oh, that was a bug. Thank you for fixing it.
